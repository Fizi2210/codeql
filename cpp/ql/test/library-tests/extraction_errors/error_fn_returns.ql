import cpp

from Function fn
where fn.getType() instanceof ErroneousType or not exists(fn.getType())
select fn
