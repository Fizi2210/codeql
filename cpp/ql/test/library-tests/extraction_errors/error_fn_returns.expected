| test.cpp:14:10:14:12 | fn2 |
